src/components/App.js

```diff
@@ -1,13 +1,46 @@
+import { useEffect, useState } from 'react'
 import styled from 'styled-components/macro'
 import MealForm from './MealForm/MealForm'
+import Button from './Button/Button'
+import MealEntry from './MealsEntry/MealsEntry'
+
+import loadFromLocal from '../lib/LoadFromLocal'
+import saveToLocal from '../lib/saveToLocal'
 
 export default function App() {
+  const [mealList, setMealList] = useState(loadFromLocal('mealList') ?? {})
+  const [currentPage, setCurrentPage] = useState('PlanMealsPage')
```

Comment:
Hey Anja, that looks really great so far...! 😀 
Wow, you managed the conditional rendering... i skipped this task... 
Do you plan to use Router for navigation in further steps? To me it was way easier, than using conditional rendering...

Reply:
Up to now, router has not been absolutely necessary. But I will use it at the latest when a proper navigation is installed :-)